### PR TITLE
User management migrations

### PR DESCRIPTION
**Pull request by Ayushnit**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,5 +26,4 @@ config/dev.*
 # deployments/docker-compose.dev.yml
 # telepresence log file
 telepresence.log
-
-
+.gitignore/
```
Comment (mukesh0513) [inline .gitignore]: why this?

File: go.mod [hunk 2]
```diff
@@ -3,6 +3,7 @@ module Tradeasy
 go 1.16
 
 require (
+	github.com/ClickHouse/clickhouse-go v1.4.5 // indirect
```
Comment (mukesh0513) [inline go.mod]: are we using these lib?

**Comment by mukesh0513** [comment]
remove unnecessary code.

**Comment by Ayushnit** [comment]
Closing this PR , changes to be made in latest PR

**Pull request closed without merging**